stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -48,8 +48,11 @@ public protocol DistributedActor:
     ///
     /// - Parameter identity: identity uniquely identifying a, potentially remote, actor in the system
     /// - Parameter transport: `transport` which should be used to resolve the `identity`, and be associated with the returned actor
-    static func resolve<Identity>(_ identity: Identity, using transport: ActorTransport)
-      throws -> Self where Identity: ActorIdentity
+// FIXME: move to generic identity here; depends on
+//    static func resolve<Identity>(_ identity: Identity, using transport: ActorTransport)
+//      throws -> Self where Identity: ActorIdentity
+    static func resolve(_ identity: AnyActorIdentity, using transport: ActorTransport)
+      throws -> Self
```

Comment:
this is related to https://github.com/apple/swift/pull/33767 - if we could "just" store the `ActorIdentity` we would not have to do this dance. And I don't want to create the struct in SIL, that's annoying and would be removed any moment now.

Reply:
Do you mean the FIXME is blocked? If so, mentioning that SE-309 would help with this in the FIXME would be good.

Reply:
Right, the fixme -- i'll add the note 👍